Gems/Atom/RPI/Code/Source/RPI.Public/WindowContext.cpp

```diff
@@ -103,6 +115,16 @@ namespace AZ
             AzFramework::WindowNotificationBus::Handler::BusDisconnect(m_windowHandle);
         }
 
+        void WindowContext::OnVsyncIntervalChanged(uint32_t interval)
+        {
+            if (m_swapChain->GetDescriptor().m_verticalSyncInterval != interval)
+            {
+                m_swapChain->SetVerticalSyncInterval(interval);
+
+                m_swapChain->Resize(m_swapChain->GetDescriptor().m_dimensions);
```

Comment:
I don't think you need to resize the swapchain to apply the vsync change. 
It will be used in next frame's swapchain present.

Reply:
OK I'll give this a try thanks

Reply:
This worked thanks